test/test_image.py

```diff
@@ -66,6 +68,65 @@ def test_damaged_images(self):
             with self.assertRaises(RuntimeError):
                 read_jpeg(image_path)
 
+    def test_encode_jpeg(self):
+        for img_path in get_images(IMAGE_ROOT, ".jpg"):
+            dirname = os.path.dirname(img_path)
+            filename, _ = os.path.splitext(os.path.basename(img_path))
+            write_folder = os.path.join(dirname, 'jpeg_write')
+            expected_file = os.path.join(
+                write_folder, '{0}_pil.jpg'.format(filename))
+            img = read_jpeg(img_path)
+
+            # PIL sets jpeg quality to 75 by default
+            jpeg_bytes = encode_jpeg(img, quality=75)
+            with open(expected_file, 'rb') as f:
+                pil_bytes = f.read()
+                pil_bytes = torch.as_tensor(list(pil_bytes), dtype=torch.uint8)
+            self.assertTrue(jpeg_bytes.equal(pil_bytes))
```

Comment:
I propose we test for contiguous and non-contiguous tensors as well
```suggestion
            with open(expected_file, 'rb') as f:
                pil_bytes = f.read()
                pil_bytes = torch.as_tensor(list(pil_bytes), dtype=torch.uint8)
                
            for src_img in [img, img.contiguous()]:
                # PIL sets jpeg quality to 75 by default
                jpeg_bytes = encode_jpeg(src_img, quality=75)
                self.assertTrue(jpeg_bytes.equal(pil_bytes))
```